tests/python/contrib/test_hexagon/test_2d_physical_buffers.py

```diff
@@ -84,6 +83,12 @@ def target_host(target):
     return tvm.target.Target(target, host=host)
 
 
+# Disabling redefined-outer-name for the whole file as there isn't any easy
+# solution yet to refactor tvm.testing.fixture fixtures that avoid redefining
+# outer variable names
+# pylint: disable=redefined-outer-name
```

Comment:
Does it not work if we just wrap the two fixtures?

Reply:
Argh, sorry, I just looked again and see this is a bit more rework than I originally thought.